pilot/pkg/bootstrap/validation.go

```diff
@@ -112,9 +112,9 @@ func (s *Server) initConfigValidation(args *PilotArgs) error {
 	}
 
 	if validationWebhookConfigName.Get() != "" {
-		s.addStartFunc(func(stop <-chan struct{}) error {
-			go whController.Start(stop)
-			return nil
+		s.leaderElection.AddRunFunction(func(stop <-chan struct{}) {
+			log.Infof("Starting validation controller")
```

Comment:
Does this mean: only one validator will be working?

Reply:
no, this is the part that modifies the webhook. all instances will serve the /validate endpoint

Reply:
ic